src/main/scala/rocks/muki/graphql/codegen/ScalametaUtils.scala

```diff
@@ -0,0 +1,13 @@
+package rocks.muki.graphql.codegen
+
+import scala.meta._
+
+object ScalametaUtils {
+  def typeRefOf(typeTerm: String): Type.Ref = {
```

Comment:
This is needed because on scalameta `4.0.0-RC1`,
```
Type.Name("a.b.c").syntax => `a.b.c`
```
So the old way does not work anymore. This is the suggested solution for now.
```
@ q"""
  package ${"a.b".parse[Term].get.asInstanceOf[Term.Ref]} {}
  """.syntax res4: String = "package a.b"
```

Reply:
Can you add some links why this is the case? Also adding these information to the scaladoc would be nice :smiley: . 
This surely looks not very robust :grimacing: